test/test_mixeddofhandler.jl

```diff
@@ -649,7 +649,7 @@ function test_vtk_export()
     sdh_tri = SubDofHandler(dh, Set(2))
     add!(sdh_tri, :u, ip_tri)
     close!(dh)
-    u = collect(1:ndofs(dh))
+    u = collect(1.:ndofs(dh))
```

Comment:
So Int types won't work after this? Perhaps you can use `float(T)` when computing the type.

Reply:
I would argue that integer types should not work here, because we assume that the solutions are in some continuous space.

Reply:
How to proceed?